src/index.html

```diff
@@ -1,13 +1,66 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content=
+      "width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect"
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link 
+      href="https://fonts.googleapis.com/
+css2?family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img 
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="imac"
+      >
+      </img>
+      <h2 class="card__headline">
+        Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+      </h2>
+      <p class="card__code">Код товара: 195434</p>
+      <div class="card__stars-feedback-align">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__feedback">Отзывов: 5</div>
+      </div>
+      <div class="card__price card__price--align">
+        <div class="card__price card__price--title">
+          Цена:
+        </div>
+        <div class="card__price card__price--main">
+          69 999 грн
+        </div>
+      </div>
+      <a
+        class="card__button" 
+        href="https://www.freecodecamp.org/" 
```

Comment:
```suggestion
        class="card__button"
        href="https://www.freecodecamp.org/"
```
не оставляй лишние пустые пробелы в конце